scripts/Muon/GUI/Common/muon_workspace.py

```diff
@@ -0,0 +1,164 @@
+# pylint: disable=F0401
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.simpleapi import mtd
+from mantid import api
+from mantid.api import Workspace, WorkspaceGroup
+
+
+def add_directory_structure(dirs):
+    """
+    create the nested WorkspaceGroup structure in the ADS specified by the
+    stored directory attribute.
+    dirs = ["dir1", "dir2"]
+    """
+    if not dirs:
+        return
+    if len(dirs) > len(set(dirs)):
+        raise ValueError("Group names must be unique")
+
+    for directory in dirs:
```

Comment:
Could I have an example of what a `directory` is? (might be useful in the string at the top).